src/LoggerConfigurator.cpp

```diff
@@ -0,0 +1,180 @@
+
+// LoggerConfigurator - reading strings for configuring the Logger
+//
+// Author  : Klaus-Dieter Gundermann
+// Created : 30.06.2020
+//
+// For all support, instructions and copyright go to:
+// http://e2guardian.org/
+// Released under the GPL v2, with the OpenSSL exception described in the README file.
+
+#ifdef HAVE_CONFIG_H
+#include "e2config.h"
+#endif
+
+#include <algorithm>
+// #include <string>
+#include "LoggerConfigurator.hpp"
+#include "String.hpp"
+
+// -------------------------------------------------------------
+// --- Constructor
+// -------------------------------------------------------------
+LoggerConfigurator::LoggerConfigurator(Logger *logger) {
+    _logger = logger;
+};
+
+LoggerConfigurator::~LoggerConfigurator() {
+    _logger = NULL;
+};
+
+const String LoggerConfigurator::Prefix("log_");
+
+// -------------------------------------------------------------
+// --- Public Functions
+// -------------------------------------------------------------
+
+bool LoggerConfigurator::configure(LoggerSource source, const std::string option) {
```

Comment:
Codacy found an issue: [Function parameter 'option' should be passed by const reference.](https://app.codacy.com/manual/numsys/e2guardian/pullRequest?prid=6238852)